redhat-appstudio/remote-secret
# Add monitoring setup for RS

### What does this PR do?
Add monitoring setup (script + GF + Prometheus kustomizes) for RS

### Screenshot/screencast of this PR

![image](https://github.com/redhat-appstudio/remote-secret/assets/1651062/0a167c78-3643-4fdb-99bb-d0f424ce644e)


### What issues does this PR fix or reference?

https://issues.redhat.com/browse/SVPI-575

### How to test this PR?

  Run `/hack/deploy-minikube-monitoring.sh`
 Navigate to GF dashboard, verify any data there
